packages/jest/index.ts

```diff
@@ -4,7 +4,8 @@ import * as path from "path";
 import * as process from "process";
 import getCacheKeyFunction from "@jest/create-cache-key-function";
 import type { Transformer, TransformOptions } from "@jest/transform";
-import { parse as parseJsonC, type ParseError } from "jsonc-parser";
+import { parse as parseJsonC } from "jsonc-parser";
+import type { ParseError } from "jsonc-parser";
```

Comment:
Prettier was failing with a SyntaxError, I guess the version doesn't understand inline type imports.